docs/index.html

```diff
@@ -3655,16 +3655,24 @@ <h2 id="adding-throttle-configuration">Adding Throttle Configuration</h2>
 </div><div class="row">
   <div class="row-height">
     <div class="col-md-5 col-sm-12 col-height  docs-primary">
-      <p><em>Added in v15.4.0.</em></p><p>When a throttle configuration is set for an action, Zapier uses it to apply throttling when the limit for the timeframe window is exceeded. It can be set at the root level and/or on an action. When set at the root level, it is the default throttle configuration used on each action of the integration. And when set in an action&apos;s operation object, the root-level default is overwritten for that action only.</p><p>To throttle an action, you need to set a <code>throttle</code> object with the following variables:</p><ol>
+      <p><em>Added in v15.4.0.</em></p><p><em>Updated in v15.6.0</em></p><p>When a throttle configuration is set for an action, Zapier uses it to apply throttling when the limit for the timeframe window is exceeded. It can be set at the root level and/or on an action. When set at the root level, it is the default throttle configuration used on each action of the integration. And when set in an action&apos;s operation object, the root-level default is overwritten for that action only. Note that the throttle limit is not shared across actions unless for those with the same key, window, and scope when &quot;action&quot; is not in the scope.</p><p>To throttle an action, you need to set a <code>throttle</code> object with the following variables:</p><ol>
```

Comment:
Should this include `limit` as part of the joined key now?

Reply:
Updated, thanks